### PR TITLE
allow puppet/archive 3.x and puppetlabs/java 3.x

### PR DESCRIPTION
**Pull request by TheMeier**
update java dependency

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -10,11 +10,11 @@
   "dependencies": [
     {
       "name": "puppet/archive",
-      "version_requirement": ">= 1.0.0 < 3.0.0"
+      "version_requirement": ">= 1.0.0 < 2.0.0"
```
Comment (bastelfreak) [inline metadata.json]: this should be `< 4.0.0`

**Comment by bastelfreak** [comment]
wah software... Can you please rebase after https://github.com/voxpupuli/puppet-kafka/pull/279 got merged? That should fix the travis issues.

**Comment by bastelfreak** [comment]
Ah actually this depends on https://github.com/voxpupuli/puppet-kafka/pull/275 :(

**Comment by TheMeier** [comment]
yeah i also stumbled across the deric/zookeeper dependencies :(

**Comment by witchbutter** [comment]
```
Warning: Module 'puppet-archive' (v3.2.1) fails to meet some dependencies:
  'puppet-kafka' (v5.3.0) requires 'puppet-archive' (>= 1.0.0 < 3.0.0)
Warning: Module 'puppetlabs-java' (v3.3.0) fails to meet some dependencies:
  'puppet-kafka' (v5.3.0) requires 'puppetlabs-java' (>= 1.4.2 < 3.0.0)
Warning: Module 'puppetlabs-stdlib' (v5.1.0) fails to meet some dependencies:
  'puppet-kafka' (v5.3.0) requires 'puppetlabs-stdlib' (>= 4.22.0 < 5.0.0)
```
This module seems to be behind on multiple dependencies even though it seems to work with all of them except java.

**Comment by bastelfreak** [comment]
yes it works. I think we should somehow get rid of the erlang module as a hard dependency.

**Comment by dhoppe** [comment]
This pull request is obsolete and has been solved by #284.

**Pull request closed without merging**